### ISSUE TITLE
Netflow IPFIX spikes in traffic, with Mikrotik CCR

### ISSUE THREAD

**Issue by dmayan** [pavel-odintsov/fastnetmon#638]
Hi,

I have installed Fastnetmon 1.1.3 on a Debian 8.6 box with Grafana. My routers are Mikrotik CCR1036 running ROS 6.37.4 (Bugfix).

As you can see in the JPEG, I'm seeing huge spikes in traffic that tend to dampen with time. If I restart fastnetmon, the spikes reappear and, again, dampen within a few hours.

I have attached the config, a capture of the netflow traffic from the router and a grafana capture showing the spikes.

Many thanks, Diego

Edit: adding netflow config of my router:

`/ip traffic-flow
set active-flow-timeout=1m cache-entries=8M enabled=yes interfaces=sfp-sfpplus2
/ip traffic-flow target
add dst-address=172.16.2.129 version=ipfix
`

[fastnetmon.conf.txt](https://github.com/pavel-odintsov/fastnetmon/files/851496/fastnetmon.conf.txt)
[netflow_ipfix.zip](https://github.com/pavel-odintsov/fastnetmon/files/851491/netflow_ipfix.zip)
![spikes](https://cloud.githubusercontent.com/assets/23533736/24058379/aae99cb6-0b29-11e7-9640-713cd1cf1478.jpeg)

**Comment by pavel-odintsov** [comment]
Hello!

First of all I would recommend this article: https://fastnetmon.com/2017/01/20/netflow-flows-duration/ 

Then I have some concerns about average_calculation_time = 12 Mikrotik ignores active/inactive timeout settings from configuration. I think 30 seconds is pretty safe value for it.

But if you could provide pcap dump with netflow for 5-10 minutes to me to pavel.odintsov@gmail.com I could check actual timeout settings and overall netflow settings.

**Comment by pavel-odintsov** [comment]
Also as mentioned here https://github.com/pavel-odintsov/fastnetmon/issues/620 your software has issues with netflow:

```
I notice a fix in RouterOS 6.38 changelog:
*) traffic-flow - fixed flow sequence counter and length;
```

Please upgrade to this version.

**Comment by dmayan** [comment]
Pavel, thanks for your reply. I choose 12 seconds based on that article, but using a 30 sec capture. I will take a 10 minutes capture and send it to you.

I'm awaiting my boss aproval to test with 6.38.5, but he is a little worried about using non-bugfix releases on a production router.

Thanks again, Diego
